docs/reference/cat/pending_tasks.asciidoc

```diff
@@ -11,8 +11,7 @@ console. They are _not_ intended for use by applications. For application
 consumption, use the <<cluster-pending,pending cluster tasks API>>.
 ====
 
-Returns cluster-level changes that have not yet been executed, similar to the
-<<cluster-pending, pending cluster tasks>> API.
+Returns <<cluster-state-publishing,cluster-level changes>> that have not yet been executed.
```

Comment:
this link might be right, but it's very confusing to me. a cluster-level change can be as simple as creating an index, but nowhere on this page does it talk about what is captured in the cluster state. we might just want to add examples that are clarifying rather than linking to this wall of text

Reply:
IMO, the cluster state publishing doc is _as close as we get_, although you're totally right it is quite an overview, so would prefer we link to it and then fix up the linked page.

Reply:
My preference would be to leave this link out because I think it would confuse people more than it would help them, but this isn't blocking